src/main/controllers/CrownWarnedListController.ts

```diff
@@ -50,7 +49,7 @@ export default class CrownWarnedListController {
                 publishedDate: publishedDate,
                 publishedTime: publishedTime,
                 version: searchResults['document']['version'],
-                provenance: metaData.provenance,
+                provenance: metaData['provenance'],
```

Comment:
I think this can be changed back to .provenance (It provides some type safety over the ['provenance'] version)